lib/cli/versions.json

```diff
@@ -0,0 +1,16 @@
+{
+  "@storybook/angular": "6.0.0-alpha.37",
```

Comment:
What's the idea here? Can you explain where this going?

Reply:
I think the goal is to use this file instead of `devDependencies` field of `package.json` of `@storybook/cli`, see here: https://github.com/storybookjs/storybook/pull/10447/files#diff-82f09434813e71512ceb5d2a5662c331R12

Reply:
correct!